docs/native-modules-setup.md

```diff
@@ -177,4 +177,24 @@ To make sure that everything is working, you'll want to try building `MyLibrary`
 
 ### Next Steps
 
-You have now created the scaffolding to build a native module or view manager. Now it's time to add the business logic to the module - follow the steps described in the [Native Modules](native-modules.md) and [View Managers](view-managers.md) documents.
\ No newline at end of file
+You have now created the scaffolding to build a native module or view manager. Now it's time to add the business logic to the module - follow the steps described in the [Native Modules](native-modules.md) and [View Managers](view-managers.md) documents.
+
+### Testing the module before it gets published
+
+#### Option 1: Create a new test app
+1. Follow the [getting started guide](getting-started.md) to create a new React Native Windows app.
+2. Run `npm i <module-local-path> --save` to install the local module.

```

Comment:
Can you include an example of what this path looks like, assuming the steps above referencing `MyLibrary`?